macOS/DuckDuckGo/NavigationBar/View/AddressBarButtonsViewController.swift

```diff
@@ -397,21 +395,16 @@ final class AddressBarButtonsViewController: NSViewController {
 
         if featureFlagger.isFeatureOn(.aiChatSidebar),
            aiChatMenuConfig.openAIChatInSidebar,
+           !isTextFieldEditorFirstResponder,
            case .url = tab.content,
            behavior == .currentTab {
-
-            if !aiChatSidebarPresenter.isSidebarOpen(for: tab.uuid),
-               isTextFieldEditorFirstResponder,
-               let value = textFieldValue,
-               let query = AIChatAddressBarPromptExtractor().queryForValue(value) {
-                // If sidebar is not open and the address bar is in focus and has viable query use it to pass as a prompt to sidebar
-                let prompt = AIChatNativePrompt.queryPrompt(query, autoSubmit: true)
-                aiChatSidebarPresenter.presentSidebar(for: prompt)
-            } else {
-                // Otherwise just toggle the sidebar
-                aiChatSidebarPresenter.toggleSidebar()
-            }
```

Comment:
1. Logic extracted to `func toggleAIChatSidebar(for tab: Tab)`
2. Pixel firing added
3. First part of the `if` check removed due to simplification of "Ask Duck.ai" no longer opening sidebar with prompt but always a full tab Duck.ai